packages/grid/test/selection.test.js

```diff
@@ -509,4 +510,52 @@ describe('multi selection column', () => {
 
     expect(grid.selectedItems).to.eql(grid.items);
   });
+
+  it('should select items on mouse drag when selectRowsByDragging is enabled', () => {
+    grid.items = ['foo', 'bar', 'baz', 'qux', 'quux', 'corge'];
+    flushGrid(grid);
+    rows = getRows(grid.$.items);
+
+    selectionColumn.selectRowsByDragging = true;
+
+    const sourceCell = getCellContent(getRowCells(rows[0])[0]);
+    const sourceCellRect = sourceCell.getBoundingClientRect();
+
+    const targetCell = getCellContent(getRowCells(rows[3])[0]);
+    const targetCellRect = targetCell.getBoundingClientRect();
+
+    const clock = sinon.useFakeTimers();
+
+    fire('track', { state: 'start', y: sourceCellRect.y, dy: 0 }, { node: sourceCell });
+
+    for (let currentY = sourceCellRect.bottom + 1; currentY < targetCellRect.y; currentY++) {
+      const currentCell = grid._cellFromPoint(sourceCellRect.x, currentY)?._content;
+      if (currentCell) {
+        const currentCellRect = currentCell.getBoundingClientRect();
+        fire(
+          'track',
+          { state: 'track', y: currentCellRect.y, dy: currentCellRect.y - sourceCellRect.y },
+          { node: currentCell },
+        );
+        clock.next();
+      }
+    }
```

Comment:
Looks like this could be simplified by just iterating the rows to select, and in each iteration get the selection column's cell and fire the track event on it.

Reply:
Done